src/components/BulkEnrollmentModal/index.jsx

```diff
@@ -71,25 +75,45 @@ class BulkEnrollmentModal extends React.Component {
         this.props.onSuccess();
       })
       .catch((error) => {
+        const { customAttributes } = error;
+        // fall back to the error.message instead of showing no information but prefer the
+        // cleaner info from customAttributes
+        const customErrMessage = truncateString(
+          customAttributes ? (customAttributes.httpErrorResponseData || error.message) : error.message,
```

Comment:
this fields comes from the interceptors here, https://github.com/edx/frontend-platform/blob/00d109d12ab207adabd8ecb2b4c8883db00d7e17/src/auth/utils.js#L61 but it's not documented! So pushing in fedx to write/review that doc somewhere to make this official